File: PHP PowerPoint Examples/ImagesApiDemo/getSlideImages.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ImagesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ImagesApi($configuration);

$name = "GetSlideImages.pptx";
$password = null;
$folder = "input";
$storage = null;
$slideIndex = 0;
$result = $apiInstance->getSlideImages($name,$slideIndex, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/ImagesApiDemo/getSlideImages.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.images_api.ImagesApi(configuration)

name = "GetSlideImages.pptx"
password = ""
storage = ""
folder = "input"

slideIndex = 0

result = api.get_slide_images(name=name, slide_index=slideIndex,
                              password=password, storage=storage,
                              folder=folder)
